index.js

```diff
@@ -80,7 +82,28 @@ module.exports = {
   },
 
   _getAllD3Modules() {
-    return d3DepsForPackage('d3', this.parent.nodeModulesPath, this.ui);
+    var d3Deps = d3DepsForPackage('d3', this.parent.nodeModulesPath, this.ui);
+
+    var pluginNames = Object.keys(this.parent.dependencies()).filter(isD3Dependency);
```

Comment:
From https://github.com/d3/d3/wiki/Plugins, it seems like almost all plugins are `d3-`prefixed, also, from https://bost.ocks.org/mike/d3-plugin/ the convention is `First, pick a name for your plugin. Start with the prefix “d3-”`. 
But I agree, the developpers would not follow this convention.
What do you think of doing both ? I mean let ember-d3 discover when they are named `d3-`* and otherwise search in the config ?

The question is: Can we assume that all package named `d3-*` are always a d3 plugin ?